### PR TITLE
documentation: ZENKOIO-56_improve_oob-md-ingestion_docs

### PR DESCRIPTION
**Pull request by wabernat**
This PR 

Added several descriptions of out-of-band updates to documentation.
simplified verb mood in headlines; renamed files to suit.
recognized use cases not involving replication.
added exposition to the introduction.

We need it because the words "out-of-band" did not appear enough before, sowing doubt, confusion, and fear. 

Fixes #ZENKOIO-56

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Review by JianqinWang — APPROVED** [review]
minor nitpick: PR commit message and PR title should be renamed to `oob-nfs-ingestion` since most times `md-ingestion` refers to RING with S3C backend

**Comment by wabernat** [comment]
yeah, but I had to leave it strategically vague for SMB/CIFS as well. 

You remind me to go hunting for S3C stuff, though.

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
-services) store data as files in buckets. Administrators are free to
-configure and name buckets as they please—to the admin, buckets are
-essentially tenants aggregated in a cloud storage deployment. From an
-end user’s perspective, a bucket simply appears as a networked storage
-location, for example, “accounting,” “bioinformatics-lab,”
-“daily-video-backup,” or any other organizational unit that makes
-sense.
+Fundamental building blocks of Zenko’s replication framework are the bucket, the
+location, and the cloud service. End users (people or services) store data as
+files in buckets. Administrators are free to configure and name buckets as they
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/Conceptual_Framework.rst:9]: Maybe this is a nitpick of `files in buckets` but might be better as `files as objects in buckets`?

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
-(expiration and transition) and other tools to manage data easily in a
-multi-cloud environment.
\ No newline at end of file
+Zenko manages the complexities and discontinuities between the native Amazon S3
+cloud storage protocol and other popular cloud protocols, such as Google Cloud
+and Microsoft Azure storage, and manages a unified namespace to retrieve data
+seamlessly from all managed cloud locations.
+
+To do this, Zenko maintains its own namespace, which it uses to manage and track
+objects in other namespaces. For example, a cloud might hold a bucket in a dozen
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/Conceptual_Framework.rst]: I'm not sure I understand `a cloud might hold a bucket in a dozen places`

File: docs/docsource/operation/Introduction/supported_clouds+services.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Zenko replicates and manages data from one or many sources to one or several
+destinations. Sources can be clouds or services, and targets can be of several
+types of cloud backend.
+
+Supported Sources
+-----------------
+
+Zenko supports multiple-cloud storage operations natively using the Amazon
+Simple Storage Service (S3) protocol. It can import data from many S3-compatible
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/supported_clouds+services.rst]: I think maybe `import data` gives the wrong idea here.

File: docs/docsource/operation/Introduction/supported_clouds+services.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+*  Amazon Simple Storage Service
+*  Scality RING with S3 Connector
+*  Wasabi Hot Cloud Storage
+*  Ceph RADOS Gateway
+
+In addition to its compatibility with S3 cloud frontends, Zenko can ingest data and
+metadata from servers using the following protocols:
+
+* Scality RING with sproxyd
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/supported_clouds+services.rst:26]: This should be S3C not sproxyd.

**Review by ssalaues — CHANGES_REQUESTED** [review]

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst:9]: As an end user, I store a file to a server somewhere. The rest, to the end user, is opaque. The "server" I see is in fact a bucket (yes?). The bucket handles the file as an object? As part of an object?

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst]: if I change "hold a bucket in a dozen places" to "distribute a bucket's contents over several disks." does that work?

File: docs/docsource/operation/Introduction/supported_clouds+services.rst (hunk 3 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/supported_clouds+services.rst]: Would "ingest" be more palatable? 
Ultimate vagueness: "handle"?

File: docs/docsource/operation/Introduction/supported_clouds+services.rst (hunk 4 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/supported_clouds+services.rst:26]: deleted (mentioned in the previous bullet list)

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 1 above)
Comment (ssalaues) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst:9]: Yeah, you're right. I think this one is fine going back over it. Marking as resolved.

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 2 above)
Comment (ssalaues) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst]: Ah, I see your intent now. Yeah either that or something like "a cloud might spread a bucket's contents across several servers".  Either way gets the point across.

File: docs/docsource/operation/Introduction/supported_clouds+services.rst (hunk 3 above)
Comment (ssalaues) [reply docs/docsource/operation/Introduction/supported_clouds+services.rst]: Yeah "handle" might be the best in this situation.

**Review by ssalaues — APPROVED** [review]

**Pull request merged**